[2024-10-19T14:29:28.765+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-19T14:29:29.077+0000] {taskinstance.py:2602} INFO - Dependencies not met for <TaskInstance: tecnica_postgres.Crear_tabla manual__2024-10-19T14:27:35.124841+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2024-10-19T14:29:29.097+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
